netbox/circuits/tables/circuits.py

```diff
@@ -76,6 +77,10 @@ class CircuitTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable):
     commit_rate = CommitRateColumn(
         verbose_name=_('Commit Rate')
     )
+    distance = columns.TemplateColumn(
```

Comment:
It's probably worth creating a custom column for this, as identical template code is also defined for WirelessLinkTable. (This new column could likely also be used for `length` on CableTable.)